addons/website_slides_survey/static/src/xml/website_slide_upload.xml

```diff
@@ -1,29 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<template>
-    <t t-name="website.slide.upload.modal.certification">
```

Comment:
A bit the same comment, we usually don't remove templates and files in stable.

But as that file was not referenced in the manifest anyway, should be fine to keep this commit :+1: